src/main/java/edu/wpi/grip/core/operations/opencv/MinMaxLoc.java

```diff
@@ -0,0 +1,75 @@
+package edu.wpi.grip.core.operations.opencv;
+
+import com.google.common.eventbus.EventBus;
+import edu.wpi.grip.core.InputSocket;
+import edu.wpi.grip.core.OutputSocket;
+import edu.wpi.grip.core.SocketHint;
+import org.bytedeco.javacpp.DoublePointer;
+import org.bytedeco.javacpp.opencv_core.Mat;
+import org.bytedeco.javacpp.opencv_core.Point;
+import org.bytedeco.javacpp.opencv_core;
+
+/** Operation to call {@link opencv_core#minMaxLoc} */
+public class MinMaxLoc implements CVOperation {
+
+    private final SocketHint<Mat>
+            srcInputHint = new SocketHint<Mat>("src", Mat.class, Mat::new, SocketHint.View.NONE, null),
+            maskInputHint = new SocketHint<Mat>("mask", Mat.class, Mat::new, SocketHint.View.NONE, null);
+
+    private final SocketHint<Number>
+            minValOutputHint = new SocketHint<Number>("minVal", Number.class, 0),
+            maxValOutputHint = new SocketHint<Number>("maxVal", Number.class, 0);
+
+    private final SocketHint<Point>
+            minLocOutputHint = new SocketHint<Point>("minLoc", Point.class, Point::new, SocketHint.View.NONE, null),
+            maxLocOutputHint = new SocketHint<Point>("maxLoc", Point.class, Point::new, SocketHint.View.NONE, null);
+
+    @Override
+    public String getName() {
+        return "Min Max Location";
+    }
+
+    @Override
+    public String getDescription() {
+        return "Finds the global minimum and maximum in an a single chanel Matrix.";
+    }
+
+    @Override
+    @SuppressWarnings("unchecked")
+    public InputSocket<?>[] createInputSockets(EventBus eventBus) {
+        return new InputSocket[] { new InputSocket(eventBus, srcInputHint), new InputSocket(eventBus, maskInputHint) };
+    }
+
+    @Override
+    @SuppressWarnings("unchecked")
+    public OutputSocket<?>[] createOutputSockets(EventBus eventBus) {
+        return new OutputSocket[] {
+                new OutputSocket(eventBus, minValOutputHint),
+                new OutputSocket(eventBus, maxValOutputHint),
+                new OutputSocket(eventBus, minLocOutputHint),
+                new OutputSocket(eventBus, maxLocOutputHint),
+        };
+    }
+
+    @Override
+    public void perform(InputSocket<?>[] inputs, OutputSocket<?>[] outputs) {
+        final Mat src = (Mat) inputs[0].getValue();
+        Mat mask = (Mat) inputs[1].getValue();
+        if (mask.empty()) mask = null;
+        final DoublePointer minVal = new DoublePointer(1);
```

Comment:
Because they aren't Double Pointers. They aren't Numbers. We need the raw numerical values. We have no way to handle Double Pointers.